src/controllers/raid.js

```diff
@@ -360,8 +363,8 @@ class Raid extends Controller {
 					jobs.push(work)
 					this.addDiscordCache(cares.id)
 				}
-				return jobs
 			}
+			return jobs
```

Comment:
looks like the most important line in the pr. 

was raid not returning all jobs?

Reply:
Correct only first egg returned (issue created)